tool/gravity/cli/status.go

```diff
@@ -284,6 +285,7 @@ func printStatusText(cluster clusterStatus) {
 }
 
 func printClusterStatus(cluster statusapi.Cluster, w io.Writer) {
+	fmt.Fprintf(w, "SELinux support:\t%v\n", formatSELinuxStatus(cluster.SELinux))
```

Comment:
Nit: I would only display this if selinux is enabled. Otherwise it's gonna be noise most of the time.